packages/measure/README.md

```diff
@@ -0,0 +1,32 @@
+# Install
+
+Make sure that `pub` is on your path. Then run:
+installed.
+```shell
+pub global activate measure
+```
+
+# Run
+First, make sure that `dart` and `flutter` is available in your command line.
+
+Then, connect a **single** iPhone, run a Flutter app on it, and
+```shell
+# assuming that you're in this directory
+measure ioscpugpu new -u resources/TraceUtility -t resources/CpuGpuTemplate.tracetemplate
```

Comment:
Is there some way we can avoid packaging a native binary with this?

Reply:
Not that I know of... This is probably due to my lack of iOS/Xcode/instruments knowledge. I'd happy to remove this binary if someone can parse the trace without using this binary.

Reply:
Just discussed with Dan offline. We can use CIPD to serve the binary instead of uploading it to git.

Reply:
Is CIPD accessible for anybody? Or is this packages only meant for "internal" use?

Reply:
We have both a public and an internal one.  This should be fine for public.  Public is at https://chrome-infra-packages.appspot.com/p/flutter, internal is https://chrome-infra-packages.appspot.com/p/flutter_internal

Reply:
(public is readable by all)

Reply:
Done.